modules/check_in/app/controllers/check_in/v2/sessions_controller.rb

```diff
@@ -33,7 +33,7 @@ def create
       end
 
       def permitted_params
-        params.require(:session).permit(:uuid, :dob, :last_name, :check_in_type)
+        params.require(:session).permit(:uuid, :dob, :last_name, :check_in_type, :facility_type)
```

Comment:
We might not need this as the front-end planning to send the facilitiy_type as a query parameter.

Reply:
as we discussed, this will be needed to POST to be distinguished for OH